scene/gui/popup_menu.cpp

```diff
@@ -1275,7 +1275,13 @@ int PopupMenu::get_current_index() const {
 
 void PopupMenu::set_item_count(int p_count) {
 	ERR_FAIL_COND(p_count < 0);
+	int prev_size = items.size();
 	items.resize(p_count);
+	if (prev_size < p_count) {
+		for (int i = prev_size; i < p_count; i++) {
+			items.write[i].id = i;
+		}
+	}
```

Comment:
I would add a new line at the beginning and end of this chunk of code, just to make it stand out better from the rest of the function.

Reply:
Sure